packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -2026,11 +2026,12 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
  
  ## Provide a k8s version to auto dashboard import script example: kubeTargetVersionOverride: 1.16.6
  ##
-@@ -76,8 +334,18 @@
+@@ -76,8 +334,19 @@
  
  ##
  global:
```

Comment:
I think you are missing `global.cattle.clusterId` and `global.cattle.clusterName`